handbook/engineering/web/index.md

```diff
@@ -1,5 +1,9 @@
 # Web team
 
+<div style="text-align: center; margin-bottom: 1.5rem">
```

Comment:
I think it would look better if we floated the logo to the left next to the text but I see that all teams have done it this way 🤷

Reply:
I changed it to the left alignment :)